README.md

```diff
@@ -97,6 +97,12 @@ To read from `stdin` and output the result to `stdout` use the `--stdin` flag.
 $ cat package.json | sort-package-json --stdin
 ```
 
+This can, for instance, be used to generate a diff before changing `package.json`.
+
+```bash
+$ ( PKG="./package.json" ; cat "${PGK?}" | pnpm sort-package-json --stdin | diff "${PKG?}" - ; )
```

Comment:
```suggestion
$ ( PKG="./package.json" ; cat "${PGK?}" | sort-package-json --stdin | diff "${PKG?}" - ; )
```

Reply:
will fix the typo too, while i'm at it (PGK instead of PKG :stuck_out_tongue_closed_eyes: )